### PR TITLE
Improve clarity in instructions

### PR DESCRIPTION
**Pull request by ChloeConnor**
- additional details in readme
- update data to 2021

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,30 +45,30 @@ We expect a `results.csv` file following this format, ordered by `Continuity` (d
...
+2. We provided a CSV file for convenience, but in a production environment, what format or database would you recommend instead for this use-case (and why)?
+3. Is there anything else you would implement differently for a large-scale application in production?
 
 
 ## Submission
 
 Please write your application in either Python, Java, Scala or Kotlin, with a runnable shell script named `run-app.sh` to launch your application.
-Please put your results in `result.csv` and submit all the files in a tarball named as follows: `syft-dataeng-<firstname>-<lastname>.tar.gz`.
-It's recommended to spend approximately 1 hour on this challenge. However, you are free to elaborate as much as you want to show us your ability to deliver great software!
+Please put your results in `result.csv` and submit all the files in a zip file named as follows: `indeedflex-dataeng-<firstname>-<lastname>.zip`.
```
Comment (adrien-mogenet) [inline README.md:73]: Makes me sad, but I reckon this will help candidates who're stuck with the gmail spam filter.

**Review by adrien-mogenet — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (adrien-mogenet) [inline .gitignore:1]: newline

(https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

**Review by mattkohl-flex — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ChloeConnor) [reply README.md:73]: Happy to change it back, but thought this might be more straightforward?

Comment (adrien-mogenet) [reply README.md:73]: Yeah I don't mind, I think I'm just emotionally attached to anything POSIX, and enjoy the thought of having candidates thinking about the process to generate a tarball.

**Pull request merged**